Gemfile

```diff
@@ -58,6 +58,15 @@ gem "rack-attack" # rate-limiting
 gem "lograge" # for JSON logging
 gem "silencer" # to disable default logging in prod
 
+# mockturtle
```

Comment:
Because this is going to run in a separate process, let's make a `mockturtle/Gemfile` for it. I don't want the Rails processes to take up any more RAM than they already do.